Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -190,17 +177,11 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
         
         for i in 0 ..< barData.dataSetCount
         {
-            guard let set = barData.getDataSetByIndex(i) else { continue }
-            
-            if set.isVisible
-            {
-                if !(set is IBarChartDataSet)
-                {
-                    fatalError("Datasets for BarChartRenderer must conform to IBarChartDataset")
```

Comment:
this messaging is important and should not be removed.

Reply:
Oops, got ahead of myself. I'll fix it in the other ones too.

Reply:
@petester42 This is corrected now. Can it be merged?